File: cypress/fixtures/application.js
const create = {
  name: 'application',
  application_type: 'siel',
  export_type: 'permanent',
  have_you_been_informed: 'yes',
  reference_number_on_information_form: '42693635'
}

const submit = {
  submit_declaration: true,
  agreed_to_foi: true,
  agreed_to_declaration: true,
  foi_reason: 'No objection',
  agreed_to_declaration_text: 'I AGREE'
}

const status = (status) => {
  return { status }
}

module.exports = { status, create, submit }

File: playwright_tests/api/fixtures/application.py
create = {
    "name": "application",
    "application_type": "siel",
    "export_type": "permanent",
    "have_you_been_informed": "yes",
    "reference_number_on_information_form": "42693635",
}


submit = {
    "submit_declaration": True,
    "agreed_to_foi": True,
    "agreed_to_declaration": True,
    "foi_reason": "No objection",
    "agreed_to_declaration_text": "I AGREE",
}


def status(status):
    return {"status": status}
